### PR TITLE
Upgrade Ruby and Rails

### PR DESCRIPTION
**Pull request by ramortegui**
This is PR upgrade ruby https://github.com/shescoding/shescoding-dot-org/issues/127  and rails https://github.com/shescoding/shescoding-dot-org/issues/128

Were both issues tackle at the same time as per pre-requisites needed.

Some of the locked Gems has been updated, as per the update.

Tested locally, and runnign the two specs. :smile:

### CODE REVIEW

**Comment by ramortegui** [comment]
@cherchezlafemme Feel free to ping me in any case with this PR :)

**Comment by cherchezlafemme** [comment]
@ramortegui  thank you so much. I will pull branch down and test.

**Comment by ramortegui** [comment]
@cherchezlafemme I'm checking the tests, as per some Gems should be added as per the updates.

**Comment by ramortegui** [comment]
@cherchezlafemme Now should be fine.  Let me know if you need any other changes on the PR 😄

File: Gemfile [hunk 1]
```diff
@@ -16,7 +16,7 @@ gem 'bootstrap_form'
 # Use Uglifier as compressor for JavaScript assets
 gem 'uglifier', '>= 1.3.0'
 # Use CoffeeScript for .js.coffee assets and views
-gem 'coffee-rails', '~> 4.0.0'
+gem 'coffee-rails'
```
Comment (cherchezlafemme) [inline Gemfile]: we can remove this one, not used

File: README.md [hunk 2]
```diff
@@ -56,26 +56,26 @@ Yes! *She's Coding* is built by the community, for the community. We currently h
...
+    
+      
 
    ##### Set ruby version if you already have rbenv installed:
    + Change your ruby version locally inside shescoding-dot-org repository:
 
       ```
       $ cd shescoding-dot-org
-      $ rbenv local 2.3.1
+      $ rbenv local 2.5.3
```
Comment (cherchezlafemme) [inline README.md]: I used to have `rbenv` as pkg manager, this didn't work - they may have changed the naming

File: README.md [hunk 3]
```diff
@@ -56,26 +56,26 @@ Yes! *She's Coding* is built by the community, for the community. We currently h
 
 	+ Install [Ruby](https://www.ruby-lang.org/en/):
       ```
-      $ rbenv install 2.3.1
+      $ rbenv install 2.5.3
```
Comment (cherchezlafemme) [inline README.md]: I used to have `rbenv` as pkg manager, this didn't work - they may have changed the naming

**Comment by ramortegui** [comment]
@cherchezlafemme I also use rvm on mac and rvm on linux.  I updated README.md with the rbenv following the instructions on their page [rbenv](https://github.com/rbenv/rbenv).

The instruction to upgrade ruby doesn't seem different for Mac or Linux (when using rvm or rbenv).  Not sure about Windows.

Not sure if it helps, but let me know if you need something else.

**Comment by cherchezlafemme** [comment]
Thank you @ramortegui I was able to verify the changes, all works great. I had to upgrade my macOS to latest, apparently old macOS was not supported by newer ruby version. Upgrading theOS solved the issue.

**Pull request merged**